docs/_sass/_colors.scss

```diff
@@ -1,17 +1,9 @@
-$color-dark: #0B1B34;
-$color-gray1: #FAFAFA;
-$color-gray2: #ECECEC;
-$color-gray3: #C6C9CA;
-$color-green: #07d973;
-$color-pink: #F68DFE;
-$color-white: #FFFFFF;
-
-$color-light-grey-green: #C9D3C5;
-$color-gray-green: #8B9C8F;
-$color-dark-green: #1B5744;
-$color-darker-green: #002E22;
-$color-super-dark-green: #061B09;
-$color-light-blue: #8DC8FF;
-
+$color-green400: #03D47C;
+$color-white: #E7ECE9;
+$color-green-icons: #8B9C8F;
+$color-green-borders: #1A3D32;
+$color-green-highlightBG: #07271F;
+$color-green-appBG: #061B09;
+$color-blue200: #8DC8FF;
 $color-link: #5AB0FF;
 $color-link-hovered: #B0D9FF;
```

Comment:
This would be color-blue100, then we'd add `$color-link-hovered:` into main.scss

Reply:
Looks like line 17 of colors.scss - it's currently `$color-link-hovered: #B0D9FF;` but I think you can change that to `$color-blue100: #B0D9FF;`

Then we will define the link-hovered color over in `main.scss`

Reply:
@shawnborton how about `$color-link` in colors.scss

Reply:
[See my comment here](https://github.com/Expensify/App/pull/17491#discussion_r1171721309)